docs/getting-started-guides/minikube.md

```diff
@@ -279,6 +279,12 @@ $ minikube start --docker-env HTTP_PROXY=http://$YOURPROXY:PORT \
                  --docker-env HTTPS_PROXY=https://$YOURPROXY:PORT
 ```
 
+Your Virtual Machine address might be 192.168.99.100, chances are your proxy settings will prevent kubectl to directly reach it.
```

Comment:
My apologies for this late response, I was waiting for an email from github to let me know about this pull request.